### PR TITLE
Adding support for Django 1.11

### PR DESCRIPTION
**Pull request by philtuskha**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def find_version(*file_paths):
         "Development Status :: 4 - Beta",
         'Environment :: Web Environment',
         'Framework :: Django',
+        'Framework :: Django :: 1.11',
```
Comment (dsummersl) [inline setup.py]: nice I was just going to mention that - you should as django 1.10 as well (that is actually where the fix to `render_to_string` became a have-to-do thing).

Comment (philtuskha) [reply setup.py]: Yeah, that makes more sense.

File: setup.py [hunk 2]
```diff
@@ -37,6 +37,8 @@ def find_version(*file_paths):
         "Development Status :: 4 - Beta",
         'Environment :: Web Environment',
         'Framework :: Django',
+        'Framework :: Django :: 1.10',
+        'Framework :: Django :: 1.11',
```
Comment (dsummersl) [inline setup.py]: Welp, I tried to run the tests for this package and I ran into more problems. So starting with what is here: https://github.com/caktus/django-sorter/blob/develop/Makefile

I setup my ccsr package to run the sorter tests:

    python manage.py test --settings=sorter.test_settings sorter

There was an error right away about `add_to_bultins` not existing anymore (see https://docs.djangoproject.com/en/2.0/releases/1.9/#django-template-base-add-to-builtins-is-removed).

I've twiddled with these tests a little more (they still don't pass for me yet)...but I think they're close.

---

Long story short:
 * Add support for django 1.9 to this section too!
 * get the test cases to pass (I'm happy to take this on if you don't mind me pushing to your PR).

Comment (dsummersl) [reply setup.py]: k - I pushed the updates to get the test cases working (hope you don't mind about that - I figured it might be a little faster since I already got started playing with that part).

**Review by dsummersl — APPROVED** [review]
Looks like it works to me! :+1: 

When you merge this into develop I'd suggest NOT deleting this branch, so that you can go ahead and use it to make a second PR against the jazzband/django-sorter repository.

**Review by dsummersl — CHANGES_REQUESTED** [review]
Actually - I just remembered another thing. We need to update the version in some way!

**Comment by dsummersl** [comment]
I suggest you change `sorter/__init__.py` to version 0.3 (based on the docs for jazzband https://jazzband.co/about/releases#versions-and-code-names, it sounds like this is a "feature" (adding support for django 1.11)).

**Review by dsummersl — APPROVED** [review]
:100:
